plyara.py

```diff
@@ -878,6 +879,11 @@ def t_STRINGNAME_ARRAY(self, t):
         t.value = t.value
         return t
 
+    def t_STRINGNAME_LENGTH(self, t):
+        r'![0-9a-zA-Z\-_*]*'
```

Comment:
I see this is the same as the `ARRAY` regex. Do we need to worry about the case where the condition looks like this:

```
    condition:
        @
```

or for this length operator, 

```
    condition:
        !
```

Plyara accepts both and outputs like this:

```
        "condition_terms": [
            "@"
        ], 
```

While YARA complains:

```
test.yar(6): error: wrong use of anonymous string
```

If we change the regex to be a little more strict, like this:

```suggestion
        r'![0-9a-zA-Z\-_*]+'
```

Plyara raises an exception, which I think might be better behavior: 

```
plyara.ParseTypeError: Illegal character @ at line 5
```

What do you think?

Reply:
Agreed. That's a better regex.

Reply:
This paradigm should be applied to t_STRINGNAME and t_STRINGNAME_ARRAY as well. Want to merge that here or make a separate PR?

Reply:
Cool. Whatever is easiest is fine, I won't be able to tag the release until tomorrow either way.